src/Presentation/Console/Commands/InstallPrivateSdkCommand.php

```diff
@@ -99,13 +99,17 @@ protected function warmUpCache(OutputInterface $output): void
     protected function installRepository(SymfonyStyle $io): void
     {
```

Comment:
Now it's not an only a private repository. InstallAdditionalSdkBundlesCommand. And Create the description how to use and how to add the new repo.